src/python/pants/testutil/rule_runner.py

```diff
@@ -547,46 +544,3 @@ def get_stdout(self) -> str:
     def get_stderr(self) -> str:
         """Return all data that has been flushed to stderr so far."""
         return self._stderr.read_text()
-
-
-class MockConsole:
```

Comment:
It looks like this breaks a couple tests. You can bump this deprecation to dev1 - no need to spend time in the release fixing all the call sites, and that would be risky to do in a release prep PR.